CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+-   #1484 : Using scope for classes to avoid name clashes.
+
 ### Changed
 
+-   #1484 : Improving handling of `DottedName` in `_assign_lhs_variable`.
```

Comment:
You should use the base form of the verb here:
```suggestion
-   #1484 : Use scope for classes to avoid name clashes.

### Changed

-   #1484 : Improve handling of `DottedName` in `_assign_lhs_variable`.
```